app/src/main/scala/org/alephium/explorer/persistence/DBInitializer.scala

```diff
@@ -51,10 +52,21 @@ class DBInitializer(val databaseConfig: DatabaseConfig[PostgresProfile])(
       LatestBlockSchema.table,
       HashrateSchema.table,
       TokenSupplySchema.table,
-      TransactionPerAddressSchema.table
+      TransactionPerAddressSchema.table,
+      AppStateSchema.table
     )
 
-  def createTables(): Future[Unit] = {
+  def initialize(): Future[Unit] = {
+    for {
+      _ <- createTables()
+      _ <- Migrations.migrate(databaseConfig)
+      _ <- createIndexes()
```

Comment:
Do we want to put index creation before the migration?

Reply:
no, as in migrations we can add a new column, if we want to create an index on that column we need first to create it.

Reply:
Got it for this specific case. In general, the dependencies might be cyclic.